strncpy possibly not null-terminating

```
/home/user/nasa/cFS/tools/cFS-GroundSystem/Subsystems/cmdUtil/cmdUtil.c:817:17: error: ‘strncpy’ specified bound 1024 equals destination size [-Werror=stringop-truncation]
                 strncpy(sbuf, &tail[1], sizeof(sbuf));
```

https://github.com/nasa/cFS-GroundSystem/blob/5e879b9f9eb4dad353e615449d3baacb8b7fde9b/Subsystems/cmdUtil/cmdUtil.c#L817

--- a/Subsystems/cmdUtil/cmdUtil.c
+++ b/Subsystems/cmdUtil/cmdUtil.c
@@ -814,7 +814,7 @@ int main(int argc, char *argv[])
                 }
 
                 /* Copy the data over (zero fills) */
-                strncpy(sbuf, &tail[1], sizeof(sbuf));
+                strncpy(sbuf, &tail[1], sizeof(sbuf) - 1);
                 CopyData(cmd.Packet, &startbyte, sbuf, tempull);
 
                 /* Reset tail so it doesn't trigger error */

